model/client.go

```diff
@@ -2334,3 +2334,17 @@ func (c *Client) ListReactions(channelId string, postId string) ([]*Reaction, *A
 		return ReactionsFromJson(r.Body), nil
 	}
 }
+
+func (c *Client) UpdateChannelRoles(channelId string, userId string, roles string) (*Result, *AppError) {
```

Comment:
Can you use the new method here with `ResponseMetadata`? Similar to this https://github.com/mattermost/platform/blob/master/model/client.go#L495